src/MCPClient/lib/clientScripts/identify_file_format.py

```diff
@@ -159,7 +170,13 @@ def main(job, command_uuid, file_path, file_uuid, disable_reidentify):
 
 def call(jobs):
     parser = argparse.ArgumentParser(description='Identify file formats.')
+
+    # Since AM19 the accepted values are "True" or "False" since the ability to
+    # choose the command from the workflow has been removed. Instead, this
+    # script will look up in FPR what's the preferred command.
+    # This argument may be renamed later.
     parser.add_argument('idcommand', type=str, help='%IDCommand%')
```

Comment:
I've decided to repurpose `%IDCommand%` because I want to minimize the number of change since we've pretty close to the release date. The values expected now are "True" and "False" - determined in the workflow data (`workflow.json`).